libs/SalesforceRestAPI/SalesforceRestAPI.xcodeproj/project.pbxproj

```diff
@@ -258,6 +292,16 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
+		CEA8838F1C19135D008D871B /* Frameworks */ = {
+			isa = PBXFrameworksBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				CEA883A61C191461008D871B /* libSalesforceSDKCore.a in Frameworks */,
```

Comment:
So e.g. this and the `libSalesforceNetwork.a` link below should go away.  The apps will be responsible for linking all of the libraries, to avoid hard dependencies between any two libraries at compile time.